web/src/views/InvestigationDetail.vue

```diff
@@ -498,17 +505,84 @@ export default defineComponent({
       }));
     }
 
+    async function toggleNonTiledImageOverlay(pin: Pin, dataset: Dataset) {
+      if (!dataset.content) return;
+      if (!featureLayer) return; // sanity check. this should exist
+      if (!nonTiledOverlayFeature) {
+        nonTiledOverlayFeature = featureLayer.createFeature('quad');
+      }
+      const quadData = nonTiledOverlayFeature.data() || [];
+      // eslint-disable-next-line
+      const existingOverlay = quadData.find((overlay: any) => overlay.pinId === pin.id);
+      if (existingOverlay) {
+        nonTiledOverlayFeature.data(
+          // eslint-disable-next-line
+          nonTiledOverlayFeature.data().filter((overlay: any) => overlay.pinId !== pin.id),
+        ).draw();
+        nonTiledOverlayFeature.draw();
+      } else {
+        try {
+          const urlRoot = process.env.VUE_APP_API_ROOT;
+          const url = `${urlRoot}/datasets/tile_source/${dataset.id}/thumbnail.png`;
+          const image: HTMLImageElement = await getNonTiledImage(url);
+          image.crossOrigin = 'use-credentials';
```

Comment:
I am curious whether the problems we were seeing with the overlays not appearing in Chrome and Brave were due to the type narrowing if statement below, and this could remain `anonymous`. That is probably worth a check.

Reply:
As stated in another comment, the issues with chrome were due to timing. Now, both `anonymout` and `use-credentials` work. I'm not sure which one is *best* here. Is there any reason we wouldn't want to send credentials?

Reply:
I don't think there is.